### ISSUE TITLE
Video controls not working in Chrome

### ISSUE THREAD

**Issue by eric-clutch** [videojs/video.js#7660]
## Description
I'm facing the issue where my I run videoJS in my Django server. The video loads just fine, but I cannot skip to future timestamps using the navigation/slider in the video controls (nor by using player.currentTime(x)). Everytime I do so, it goes back to the start. Other controls such as playback speed, pause/play, work.

This happens ONLY in Chrome, in Safari it works correctly. However, in Safari I cannot load videos larger than 300 MB (I don't know why, any ideas?) but in Chrome I can.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Open HTML page containing the video using Chrome
2. Try to use the slider control.

## Results
Video goes back to the start frame.
### Expected
Video jumps to whatever time I had selected.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
7.10.2 using CDN
#### browsers
Chrome affected; Safari is not.
#### OSes
only tested in macOS
### plugins
None

**Comment by gkatsev** [comment]
What type of video are you playing? Is it mp4? Does your server support byte-range requests? Chrome requires them for seeking in an mp4.

**Comment by eric-clutch** [comment]
It's mp4 (h.264)

**Comment by eric-clutch** [comment]
@gkatsev Yes, that video works on Chrome, I can use the slider.

Do you have any idea what kind of tweaks do I need, or what resources I can use? I am using django in the backend.
What I'm most surprised about is Safari handling it okay but not Chrome.

**Comment by gkatsev** [comment]
yeah, doesn't seem like django supports byte range requests by default based on this, SO post https://stackoverflow.com/questions/14324250/byte-ranges-in-django

Safari and Chrome manage things a bit differently internally, so, it's possible for it to work in one browser and another another.

**Comment by okaeiz** [comment]
We have the same problem and the slider is not working in Chrome, but is alright in Firefox.
Is there any latest update on this?

**Comment by gkatsev** [comment]
Please open a new issue, if you're able to provide an example.
